src/common/cursor.c

```diff
@@ -34,6 +34,14 @@
 #include <stdlib.h>
 #include <string.h>
 
+
+/**
+ * Allocates a cursor as well as an image buffer where the cursor is rendered.
+ * If the allocation fails then the function returns NULL.
```

Comment:
This is much better, but please be sure to also use the `@return` tag to document the return value/behavior specifically, not just in the overall description of the function. For example:

https://github.com/apache/incubator-guacamole-server/blob/025fc0525f28b05be206d1471e2d8d488f97cb14/src/pulse/pulse/pulse.h#L92-L94

Reply:
:+1: Done